### PR TITLE
changed 1 to 2 in first file

### PR DESCRIPTION
**Pull request by AlinaSte**

### CODE REVIEW

**Comment by gh3rtzz** [comment]
Where is number 3?))

**Comment by AlinaSte** [comment]
waiting approval for merge

File: first_file.txt [hunk 1]
```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+3
```
Comment (gh3rtzz) [inline first_file.txt]: Nice, now change 3 to 4

Comment (AlinaSte) [reply first_file.txt]: done

**Review by gh3rtzz — APPROVED** [review]
Approved. Go ahead and merge it

**Pull request merged**